airdrops/index_v1.json

```diff
@@ -173,6 +173,16 @@
             "name": "OP",
             "icon": "optimism.svg",
             "cutoff_time": 1739491200
+        },
+        "degen2_season1": {
+            "csv_path": "airdrops/degen2_season1.csv",
+            "csv_hash": "708ae1fcbe33a11a91fd05e1e7c4fa2d514b65cb1f8d3e4ce3556e7bdd8af2f5",
+            "asset_identifier": "eip155:8453/erc20:0x4ed4E862860beD51a9570b96d89aF5E1B0Efefed",
```

Comment:
I'm thinking that maybe we should add the assets details. The code does a `get_or_create right`? I'm thinking in the case of someone that hasn't pulled the last update... then the app will break for him. There is no harm in adding them right?